src/otx/config/mmdeploy/base/base_detection_dynamic.py

```diff
@@ -0,0 +1,22 @@
+"""Detection models dynamic deploy config."""
+
+_base_ = ["./base_detection_static.py"]
+
+ir_config = dict(
+    dynamic_axes={
+        "image": {
+            0: "batch",
+            1: "channel",
+            2: "height",
+            3: "width",
+        },
+        "boxes": {
+            0: "batch",
+            1: "num_dets",
```

Comment:
`num_dets` is the only dynamic dimension that's reasonable to keep here

Reply:
Can't `batch` be also dynamic?

Reply:
In MAPI batch is always 1

Reply:
OV prioritizes multiple infer requests over batching

Reply:
Do you mean that model will get single batch in MAPI? When I checked in the OTX sides, it can get multiple batch and output multiple batch output.

Reply:
Yes, exactly, see OV adapter and ImageModel implementations in MAPI if you're interested in that. Batch != 1 is not supported there.